source/server/logger.js

```diff
@@ -0,0 +1,89 @@
+Space.Object.extend(Space.jobQueue, 'Logger', {
+
+  dependencies: {
+    log: 'log'
+  },
+
+  eventSubscriptions() {
+    return [
+      {
+        'Space.jobQueue.JobServerStarted'() {
+          this.log.info("Job Server Started");
+        }
+      },
+      {
```

Comment:
you don't need an object for each event type! you can just return an array with a single object and the event names as keys like this:

```javascript
return [{
  'Space.jobQueue.JobServerStarted'() {
    this.log.info("Job Server Started");
  },
  'Space.jobQueue.JobServerShutdown'() {
    this.log.info("Job Server shutdown");
  },
  // …
}]
```